src/pages/authen/index.tsx

```diff
@@ -0,0 +1,30 @@
+import LoginForm from '@/components/Login/LoginForm'
+import WelcomeTextBox from '@/components/Login/WelcomeTextBox'
+import React from 'react'
+
+interface Props {}
+
+const Login: React.FC<Props> = (props) => {
```

Comment:
naming Page component name as PageNamePage and use type NextPage instend of React.FC